website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -4,16 +4,16 @@ id: project-dependencies
 sidebar_label: "Project dependencies"
 description: "Reference public models across dbt projects"
 pagination_next: null
+keyword: dbt mesh, project dependencies, ref, cross project ref, project dependencies
 ---
 
 :::info Available in Public Preview for dbt Cloud Enterprise accounts
 
 Project dependencies and cross-project `ref` are features available in [dbt Cloud Enterprise](https://www.getdbt.com/pricing), currently in [Public Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). 
 
-Enterprise users can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-use-ref).
+Enterprise users can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-write-cross-project-ref).
```

Comment:
Do we have a style guideline about when we say "users" versus "customers"? Normally I think of "user" as "individual" (writing dbt code), and "customer" as the organization leveraging dbt Cloud (including significant features of the platform)
```suggestion
Enterprise customers can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-write-cross-project-ref).
```

Reply:
that's a good question and we normally user 'users' and [avoid customer](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#terms-to-use-or-avoid). specifically i noticed we use 'plans' or 'accounts' when referencing features available on certain plans. i am more than happy to raise this with the docs team to see if we can get more clarity here!

Reply:
also added link to enterprise for add'l context: https://www.getdbt.com/pricing